docs/dev/table/streaming/match_recognize.md

```diff
@@ -902,6 +902,37 @@ One has to keep in mind that in case of the `SKIP TO FIRST/LAST variable` strate
 variable (e.g. for pattern `A*`). In such cases, a runtime exception will be thrown as the standard requires a valid row to continue the
 matching.
 
+Time attributes
+---------------
+
+In order to apply some subsequent queries on top of the `MATCH_RECOGNIZE` it might be required to use [time attributes](time_attributes.html). To select those there
```

Comment:
Can you split the longer lines (maybe even in the whole file in a separate commit, or at least the newly added)? Longer lines are causing problems with resolving conflicts and make diffs less readable.